quickstarts/deploy.markdown

```diff
@@ -0,0 +1,54 @@
+Kroxylicious is a Java application based on [Netty](https://netty.io/), which means it will run anywhere you can run a JVM. (That's a lot of places!)
+To keep things simple, we've created a quick setup guide to help.
+
+<br />
+
+## Getting started
+
+#### Prerequisites
+
+To get started deploying Kroxylicious, you will need to install a Java Runtime Environment (JRE) with minimum version 17. This does not come included with the Kroxylicious JAR files.
+
+Some operating systems come with a JRE already installed. You can check what Java version you have installed by running the following command:
+
+```shell
+java -version
+```
+
+If you get an error or the command doesn't return anything, you may not have a JRE installed, or your JRE may not have been correctly added to your system's `PATH` variable.
+
+<br />
+
+#### Downloading Kroxylicious
+
+Kroxylicious can be downloaded from the [releases](https://github.com/kroxylicious/kroxylicious/releases) page of the Kroxylicious GitHub repository, or from Maven Central.
+
+In GitHub, all releases since v0.4.0 have an attached `kroxylicious-app-*-bin.zip` file. Download this, and optionally verify the contents of the package with the attached `kroxylicious-app-*-bin.zip.asc` file.
+
+<br />
+
+## Install and Run
+
+#### Installing Kroxylicious
+
+Extract the downloaded Kroxylicious Zip file into the directory you would like to install Kroxylicious in. Ensure the `kroxylicious-start.sh` and `run-java.sh` files in the `bin/` directory within the extracted folder have at least read, write, and execute (`rwx`) permissions for the owner.
```

Comment:
True. I guess I was thinking that a particularly inexperienced user might see "read and execute" and think "oh I'll go remove the write permissions then". But I think you're right, and I can omit the write permissions here.

Reply:
If we direct the user to the tgz rather than the zip we avoid needing to mention anything about the permissions.  AFAIK Windows tooling handles the tar/gz file formats.

Reply:
OH thats a good point @k-wall I didn't mention using tgz vs zip as I thought it was just my bias showing. We should also double check what permissions we ship the scripts with.

Reply:
@k-wall Only with 3rd party tools. On Windows, 7zip or similar is required to process tar archives or gzipped files. I wanted to go with the most accessibly cross-platform format, which is (unfortunately) zip.

In saying that though, have we ever tested Kroxylicious on Windows? Maybe it doesn't work and I can ignore that platform

Reply:
I've elected to leave the format as `zip` so we aren't excluding Windows users. We can always change this later if we feel we'd rather use `tar`.